src/kili/llm/presentation/client/llm.py

```diff
@@ -83,6 +86,7 @@ def export(
             project_id=ProjectId(project_id),
             asset_id_in=resolved_asset_ids,
             label_type_in=label_type_in,
+            status_in=status_in or ["LABELED", "REVIEWED", "TO_REVIEW", "ONGOING"],
```

Comment:
@baptiste-olivier je me demande pour le ONGOING, on avait viré cette valeur pour pouvoir send back des labels to queue et ne pas les avoir dans l'export pour simuler une suppression de label

Reply:
et pourquoi on met l'array par défaut ici? Ca apparait pas dans le doc du coup

Reply:
Hello @theodu , 
You can directly use the option `include_sent_back_labels` to ignore or add sent back labels.